Playbooks/playbook-Extract_Indicators_From_File_-_Generic.yml

```diff
@@ -748,5 +748,5 @@ outputs:
 - contextPath: URL.Data
   description: List of URLs that were extracted from the PDF file
   type: unknown
-tests:
-  - Extract Indicators From File - test
+tests: 
+  - No test - deprecated playbook
```

Comment:
should the description be updated too? currently there is no indication it is deprecated and what replaced it.